specs/phase0/beacon-chain.md

```diff
@@ -1308,27 +1307,27 @@ def get_attestation_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], Sequence
         attesting_balance = get_total_balance(state, unslashed_attesting_indices)
         for index in eligible_validator_indices:
             if index in unslashed_attesting_indices:
-                rewards[index] += get_base_reward(state, index) * attesting_balance // total_balance
+                rewards[index] += get_base_reward(state, total_balance, index) * attesting_balance // total_balance
```

Comment:
Instead of changing the input args on this function, can we instead put a cache on `get_total_active_balance` to get a similar gain?

Reply:
Yes we probably should, this idea of not recomputing the balance to avoid a lot of state iterations was put in place before the memoization was implemented.

Reply:
Fixed in 5c5b528
Probably a good idea to squash the changes in this PR. There's a lot of dev iteration on small things.